main.go

```diff
@@ -270,8 +282,18 @@ func enterProvider(wg *sync.WaitGroup, p provider.Provider, mergerCH <-chan []st
 				bootstrap = false
 				continue
 			}
-
 			var err error
+			createNotify := func(err error, t time.Duration) {
+				switch errors.Cause(err).(type) {
+				case *provider.AlreadyExistsError:
+					err = p.Update(output)
+					if err != nil {
+						log.Error(err)
+					}
+				default:
+					log.Errorf("updating stack err: %v at time: %v", err, t)
```

Comment:
Signature of RetryNotify:
`func RetryNotify(operation Operation, b BackOff, notify Notify) error `

There is no need log here.

Reply:
removed.